scanner/cmd/scanner/main.go

```diff
@@ -80,6 +81,11 @@ func main() {
 	if err != nil {
 		golog.Fatalf("failed to initialize logging: %v", err)
 	}
+
+	if err := continuousprofiling.SetupClient(continuousprofiling.DefaultConfig()); err != nil {
+		zlog.Error(ctx).Err(err).Msg("unable to start continuous profiling")
+	}
```

Comment:
Does this make our debug HTTP endpoints redundant? Is there any risk in running pyroscope and some calls the debug endpoint at the same time?

Reply:
If pyroscope is enabled, then the debug endpoints will not work but you can still download the profiles directly from pyroscope.

Reply:
My first reaction to this is to disable the debug endpoints *if* pyroscope is enabled. You?

Reply:
I am also thinking we don't have to do it here. Maybe capture a follow up to revisit these debug endpoints.

Reply:
I've created [ROX-28465](https://issues.redhat.com/browse/ROX-28465) to disable the pprof servers in all components if pyroscope is enabled.